_includes/parse-server/deploying-aws-bitnami.md

```diff
@@ -0,0 +1,32 @@
+## Deploying on AWS using Bitnami AMI
+
+Here are the steps:
+
+1. Log into your AWS account or create a new one [AWS Account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/)
+2. Go to AWS EC2 Dashboard
+3. Launch Instances
+4. On **Application and OS Images (Amazon Machine Image)** search for `Parse`
+5. Select **Parse Server packaged by Bitnami**
+6. Click `Continue`
+7. On **Instance Type** select `t2.micro` (is ok for testing and small projects which is Free tier eligible)
+8. Create or select an existing `key pair`. (If you create a new one click `Download Key Pair`)
+9. Click `Launch Instance`
+10. On **EC2 Dashboard** select the new created instance and click `Connect`
+11. Click `SSH Client` and follow the instructions
+
+Login into the ec2 instance we perform the following commands:
+
+```bash
+cd stack/parse
+```
+```bash
+cat config.json
+```
+
+That will show Parse configuration details, save those details for later.
+
+<!-- WIP -->
```

Comment:
Actually I haven't finish the Bitnami guide, because I was not sure if it will be published or not.. and I never did the Bitnami deployment myself, I just thought that could be useful for other users... not really sure if I have time/energy to test the Bitnami deployment and finish the guide to be honest... I will comment the link from the deploying.md file and I will decided later if I finish it...

Reply:
We'd need to at least test the guide to ensure it's correct. What you could do it split the guide off i to a separate PR, so we can merge the other changes and at a later point merge the Bitnami guide once we know it works.

Reply:
I am a bit lost now...  I did remove the Bitnami guide file, so you can merge the PR now??